### PR TITLE
CON-2234_Display certificate status information

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

**Review by shomavg — APPROVED** [review]

**Review by shomavg — APPROVED** [review]

File: src/SFA.DAS.AssessorService.Data/CertificateRepository.cs [hunk 1]
```diff
@@ -485,7 +488,7 @@ public async Task UpdateSentToPrinter(Certificate certificate, int batchNumber,
                         Status = CertificateStatus.SentToPrinter,
                         StatusAt = sentToPrinterDate,
                         CreatedBy = SystemUsers.PrintFunction
-                    });
+                    }); ;
```
Comment (cofaulco) [inline src/SFA.DAS.AssessorService.Data/CertificateRepository.cs:491]: double semi-colon

**Review by cofaulco — COMMENTED** [review]
Double semi-colon type

File: src/SFA.DAS.AssessorService.Data/CertificateRepository.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Data/CertificateRepository.cs]: Fixed

**Review by cofaulco — APPROVED** [review]

File: src/SFA.DAS.AssessorService.Application/Handlers/Staff/GetLearnerDetailHandler.cs [hunk 2]
```diff
@@ -99,15 +101,24 @@ public async Task<LearnerDetailResult> Handle(GetLearnerDetailRequest request, C
...
-                ReasonForChange = certificate?.CertificateBatchLog?.ReasonForChange
+                ReasonForChange = certificate?.CertificateBatchLog?.ReasonForChange,
+                ContactName = certificateData.ContactName,
+                ContactOrganisation = certificateData.ContactOrganisation,
+                ContactAddLine1 = certificateData.ContactAddLine1,
+                ContactAddLine2 = certificateData.ContactAddLine2,
+                ContactAddLine3 = certificateData.ContactAddLine3,
+                ContactAddLine4 = certificateData.ContactAddLine4,
+                ContactPostCode = certificateData.ContactPostCode,
+                LastUpdatedAt = new[] { certificate?.ToBePrinted, certificate?.CreatedAt, certificate.LatestChange(),
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Application/Handlers/Staff/GetLearnerDetailHandler.cs:112]: The certificate.LatestChange is exactly what you are after here - just return that

**Review by ChrisJWoodcock — COMMENTED** [review]
I authored this PR so I can't mark it request changes for some reason... but that is what is required

File: src/SFA.DAS.AssessorService.Application/Handlers/Staff/GetLearnerDetailHandler.cs (hunk 2 above)
Comment (shomavg) [reply src/SFA.DAS.AssessorService.Application/Handlers/Staff/GetLearnerDetailHandler.cs]: done.

**Review by ChrisJWoodcock — COMMENTED** [review]
I'm approving this - although I started it - so you will have to approve it instead

**Review by cofaulco — APPROVED** [review]

**Pull request merged**